exalearn-ui/src/index.html

```diff
@@ -11,6 +11,7 @@
 		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
 	</head>
 	<body class="mat-typography">
+    <header></header>
 		<app-root></app-root>
 	</body>
```

Comment:
All components and everything else should be in app-root, also usually components have the 'app-' prefix in selector.